server/models/Profile.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema(
+    {
+        firstname: {
+            type: String,
+            required: true,
+        },
+        lastname: {
```

Comment:
We generally prefer to use `camelCase` for properties in JS by convention, could you please update the model accordingly?